details.md

```diff
@@ -0,0 +1,196 @@
+### Proposed types of layers
```

Comment:
I'm super looking forward to working on this design when we get closure on the next steps for multilayer!